### PR TITLE
Add channel delete

### PR DESCRIPTION
**Pull request by MartinDelille**
I'm not really sure if I documented well the responses description/type

### CODE REVIEW

File: source/channels.yaml [hunk 1]
```diff
@@ -195,6 +195,28 @@
         '500':
           description: Could not retrieve the channel stats, the user is not a member of the channel or the channel does not belong to the team
 
+  '/teams/{team_id}/channels/{channel_id}/delete':
+    post:
+      tags:
+        - channels
+      summary: Delete a channel
+      description: Add a channel from a team.
```
Comment (jwilander) [inline source/channels.yaml]: I think you missed updating the description

File: source/channels.yaml [hunk 2]
```diff
@@ -195,6 +195,28 @@
...
+          type: string
+        - name: channel_id
+          in: path
+          description: The channel ID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: The channel ID
+          type: string
```
Comment (jwilander) [inline source/channels.yaml]: I think the response is actually an object like so:
```
{
    "id": "idgoeshere"
}
```

Comment (MartinDelille) [inline source/channels.yaml]: Ok how can I document that?

**Comment by MartinDelille** [comment]
Done

**Comment by jwilander** [comment]
Much thanks!

**Pull request merged**